pipeline/inputs/node-exporter-metrics.md

```diff
@@ -22,6 +22,21 @@ This plugin is currently only supported on Linux based operating systems\
 | scrape_interval | The rate at which metrics are collected from the host operating system | 5 seconds |
 | path.procfs     | The mount point used to collect process information and metrics        | /proc/    |
 | path.sysfs      | The path in the filesystem used to collect system metrics              | /sys/     |
+| ne.cpu.scrape\_interval       | The rate at which cpu metrics are collected from the host operating system. To enable, specify greater than 0. | 0 seconds (disabled by default) |
```

Comment:
Instead of setting to 0 would it be better to have it unset? Or does it require a value?

`Disabled by default` is not right here, I think we should say it uses the global `scrape_interval` by default (what if that is set to 0??). We do not disable the polling of metrics, we just do not configure it with an override.

Reply:
`ne.xxx.scrape_interval` will ignite to start each of timers for tied up metrics. They should be disabled -- this is because we should keep as-is or keep plugin behavior at least from behavior of first implementation of this plugin.

On Fluentd world, we can announce through deprecated warning that can be generated from config_param options. However, Fluent Bit does not have such functionality.

Reply:
Sure, I understand the implementation but the documentation is not clear from a user perspective I think. In that case the global scrape_interval is used: if you do not set a custom interval here then you will get metrics at the global interval right? The user only cares about the outcome and not what it means for the implementation - saying it is disabled suggests they won't get these metrics.

Reply:
Ah, you are right. There was lack of description for "specific intervals".
Modified in https://github.com/fluent/fluent-bit-docs/pull/1034/commits/f322923a50346594f36c0ed664473b482fa90b24.